website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -9,6 +9,23 @@ To-do:
 - use the reference doc structure for this article/split into separate articles
 --->
 
+| Configuration    | Description    | 
+|------------------|----------------|
+|[`dataset`](/reference/resource-configs/bigquery-configs#use-project-and-dataset-in-configurations)|`schema` is equivalent to a `dataset` in BigQuery.|
+|[`project`](/reference/resource-configs/bigquery-configs#use-project-and-dataset-in-configurations)|`database` corresponds to a `project` in BigQuery.|
+|[`partition_by`](/reference/resource-configs/bigquery-configs#partition-clause)|`partition_by` partitions tables. It helps in reducing query latency and cost. Partition pruning works only with literal filters, not subqueries.|
+|[`cluster_by`](/reference/resource-configs/bigquery-configs#clustering-clause)|Clustered tables in BigQuery use a user-defined column sort order to enhance query performance and reduce costs.|
+|[KMS encryption](/reference/resource-configs/bigquery-configs#managing-kms-encryption)|Key Management Service (KMS) encryption &mdash; method of encrypting data using managed encryption keys provided by cloud services like Google Cloud KMS|
```

Comment:
this might be just me but i feel like the description should describe the config? rather than explain what kms means, which is already explained in the linked bq doc:

```suggestion
|[KMS encryption](/reference/resource-configs/bigquery-configs#managing-kms-encryption)| Configure [customer managed encryption keys](https://cloud.google.com/bigquery/docs/customer-managed-encryption) for BigQuery tables using the `kms_key_name` model configuration. |

```